raiden/tests/unit/test_channelstate.py

```diff
@@ -1194,7 +1195,77 @@ def test_channel_must_never_expire_locks_with_onchain_secret():
 
     lock_amount = 10
     lock_expiration = block_number - 10
-    lock_secret = b'test_channel_must_accept_expired_locks'
+    lock_secret = sha3(b'test_channel_must_never_expire_locks_with_onchain_secret')
+    lock_secrethash = sha3(lock_secret)
+    lock = HashTimeLockState(
+        amount=lock_amount,
+        expiration=lock_expiration,
+        secrethash=lock_secrethash,
+    )
+
+    nonce = 1
+    transferred_amount = 0
+    receive_lockedtransfer = make_receive_transfer_mediated(
+        channel_state=channel_state,
+        privkey=privkey2,
+        nonce=nonce,
+        transferred_amount=transferred_amount,
+        lock=lock,
+    )
+
+    is_valid, _, msg = channel.handle_receive_lockedtransfer(
+        channel_state=channel_state,
+        mediated_transfer=receive_lockedtransfer,
+    )
+    assert is_valid, msg
+
+    assert lock.secrethash in channel_state.partner_state.secrethashes_to_lockedlocks
+
+    lock_expired = make_receive_expired_lock(
+        channel_state,
+        privkey2,
+        receive_lockedtransfer.balance_proof.nonce + 1,
+        transferred_amount,
+        lock,
+        locked_amount=0,
+    )
+
+    is_valid, msg, _ = channel.is_valid_lock_expired(
+        state_change=lock_expired,
+        channel_state=channel_state,
+        sender_state=channel_state.partner_state,
+        receiver_state=channel_state.our_state,
+        block_number=block_number,
+    )
+
+    assert is_valid, msg
```

Comment:
Hummmmm, this is really weird, the test is named `test_channel_must_never_expire_locks_with_onchain_secret` however this is asserting on the lock_expired *being accepted*.

It looks like this test and the one bellow got swapped.